### ISSUE TITLE
LDAP - from posixGroup to groupOfNames

### ISSUE THREAD

**Issue by fvanderbiest** [georchestra/georchestra#569]
geOrchestra currently ships with a default LDAP setup having groups which are `objectClass: posixGroup`. This kind of objects references users via the `memberUid` property (storing, guess what ... uids).

In the near future, we might have to change the groups' class in favor of `objectClass: groupOfNames`.
We are indeed currently facing difficulties configuring Pydio to map LDAP groups to pydio "roles", as explained here: http://pyd.io/groups-and-roles-with-ldapad-integration-draft/. Pydio expects users to have the `memberOf` property linking to the appropriate groups.
In addition, several people already reported that they prefer having full `DN` stored in the member property of `groupOfNames`-typed groups.

This would require to change geOrchestra in several places:
- security proxy config
- cas config
- ldapadmin code & config
- geonetwork config
  .. and of course would impact all current instances wishing to migrate.

See also the discussion here: https://groups.google.com/forum/#!topic/georchestra-dev/Uf5bvO_Ajug

Basically, this would mean reverting most of the changes from https://github.com/georchestra/georchestra/pull/91
